src/ckeditor/ckConfigurator.js

```diff
@@ -431,6 +433,11 @@ const ckConfigurator = (function () {
         disableNativeTableHandles: true
     };
 
+    //update default specialChars if config for it exists
```

Comment:
Just a thought, do we really need this comment? It seems it is explaining the if itself.

Reply:
This particular comment may be not that needed indeed